h263_codec_registration.src.html

```diff
@@ -0,0 +1,154 @@
+
+    This registration is non-normative.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: attribute
+        text: EncodedVideoChunkMetadata.decoderConfig; url: dom-encodedvideochunkmetadata-decoderconfig
+        for: EncodedVideoChunkType; text: key; url: dom-encodedvideochunktype-key
+        text: VideoDecoderConfig.description; url: dom-videodecoderconfig-description
+    type: dfn
+        for: EncodedVideoChunk; text: [[internal data]]; url: dom-encodedvideochunk-internal-data-slot
+        for: EncodedVideoChunk; text: [[type]]; url: dom-encodedvideochunk-type-slot
+        for: VideoEncoder; text: [[output callback]]; url: dom-videoencoder-output-callback-slot
+    type: interface
+        text: EncodedVideoChunk; url: encodedvideochunk
+        text: VideoEncoder; url: videoencoder
+    type: dictionary
+        text: VideoEncoderConfig; url: dictdef-videoencoderconfig
+        text: VideoDecoderConfig; url: dictdef-videodecoderconfig
+</pre>
+
+<pre class='biblio'>
+{
+  "H.263v1": {
+    "href": "https://www.itu.int/rec/T-REC-H.263-199603-S/en",
+    "title": "H.263: Video coding for low bit rate communication (03/96)",
+    "publisher": "ITU-T"
+  },
+  "H.263v2": {
+    "href": "https://www.itu.int/rec/T-REC-H.263-199802-S/en",
+    "title": "H.263: Video coding for low bit rate communication (02/98)",
+    "publisher": "ITU-T"
+  },
+  "H.263v3": {
+    "href": "https://www.itu.int/rec/T-REC-H.263-200501-I/en",
+    "title": "H.263: Video coding for low bit rate communication (01/2005)",
+    "publisher": "ITU-T"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string begins with the prefix `"h263."`, with a suffix of 1 character
+describing the version of the specification: `"h263.1"` for the first revision
+([[H.263v1]]), `"h263.2"` for the second revision ([[H.263v2]]), or `"h263.3"`
+for the third revision ([[H.263v3]]). As the specifications are backwards
+compatible, an encoder may encode using an earlier revision than specified, and
+a decoder may decode using a later revision than specified.
+
+`"h263.3"`, but no prior revision, may further be suffixed with a `"."` and a
+one-character profile number, as specified in Annex X.2 of [[H.263v3]]. If a
+profile is not specified, then an encoder may use any optional features and a
+decoder must support all optional features. This may further be suffixed by a
+`"."` and a two-character level, as described in Table X.2 of [[H.263v3]]. If a
```

Comment:
Are levels in H.263 inter-compatible, as in, can an encoder output a lower level than requested like we have defined for H.264?

Reply:
With the exception of level 45, each level is strictly looser than all previous levels, so an encoder outputting at, e.g., level 10 is correctly outputting at level 20 or 30 or 40. 45 is an exceptional case, and while all streams in level 45 are also valid bitstreams in all higher levels, it is not true that all valid bitstreams at lower levels are valid as level 45. Because 45 is a weird exception, but all other levels are inter-compatible, I think it's easier to leave that as implicit, rather than repeating the document's rather long-winded explanation of their intercompatibility that I've just detailed here :)